modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc

```diff
@@ -173,109 +179,103 @@ include::example$migrating.py[tag=certauth,indent=0]
 //- important: bootstrapping is now "lazy" so also discuss how to do the eager
 // checks and any impact it might have on the first op being slow?
 
-From a high-level perspective, bootstrapping and shutdown is very similar to SDK 2.
-////
-One notable difference is that the `Collection` is introduced and that the individual methods like `bucket` immediately return, and do not throw an exception.
-Compare SDK 2: the `openBucket` method would not work if it could not open the bucket.
-
-The reason behind this change is that even if a bucket can be opened, a millisecond later it may not be available any more.
-All this state has been moved into the actual operation so there is only a single place where the error handling needs to take place.
-This simplifies error handling and retry logic for an application.
-
-- discuss shutdown
-- if external state is passed in optionally, discuss
-management of this state
-- discuss the impact of 6.5 "gcccp" and cluster level queries
-- SASL by default on non-PLAIN (if applicable)
+From a high-level perspective, bootstrapping and shutdown is very similar to SDK API 2.
 
+////
+// One notable difference is that the `Collection` is introduced and that the individual methods like `bucket` immediately return, and do not throw an exception.
+// Compare SDK API 2: the `openBucket` method would not work if it could not open the bucket.
 
+// The reason behind this change is that even if a bucket can be opened, a millisecond later it may not be available any more.
+// All this state has been moved into the actual operation so there is only a single place where the error handling needs to take place.
+// This simplifies error handling and retry logic for an application.
 
-In SDK 2, you connected, opened a bucket and performed a KV op like this:
+// - discuss shutdown
+// - if external state is passed in optionally, discuss
+// management of this state
+// - discuss the impact of 6.5 "gcccp" and cluster level queries
+// - SASL by default on non-PLAIN (if applicable)
 
-[source,python]
+// In SDK API 2, you connected, opened a bucket and performed a KV op like this:
 
-----
-from couchbase_v2.cluster import Cluster as SDK2Cluster, PasswordAuthenticator
-cluster = SDK2Cluster("127.0.0.1")
-cluster.authenticate(PasswordAuthenticator("user", "pass"))
-bucket = cluster.open_bucket("travel-sample")
+// [source,python]
+// ----
+// from couchbase_v2.cluster import Cluster as SDK2Cluster, PasswordAuthenticator
+// cluster = SDK2Cluster("127.0.0.1")
+// cluster.authenticate(PasswordAuthenticator("user", "pass"))
+// bucket = cluster.open_bucket("travel-sample")
 
-get_result = bucket.get("airline_10")
+// get_result = bucket.get("airline_10")
+// ----
 
-----
-
-Here is the SDK 3 equivalent:
-
-[source,python]
+// Here is the SDK API 3 equivalent:
 
-----
-include::example$migrating.py[tag=simpleget,indent=0]
-----
+// [source,python]
+// ----
+// include::example$migrating.py[tag=simpleget,indent=0]
+// ----
 ////
+
```

Comment:
Should this be uncommented?

Reply:
I don't think so - looks like it was previously block-commented, so these are just making that explicit.